manifests/crd.yaml

```diff
@@ -1,18 +1,18 @@
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1alpha1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-  name: jobs.metacontroller.app
+  name: recipes.metacontroller.io.app
```

Comment:
recipes.dope.metacontroller.io
where metacontroller.io is the primary domain
where dope.metacontroller.io is the domain for this project